core/src/main/scala/io/finch/instances/IO.scala

```diff
@@ -0,0 +1,30 @@
+package io.finch
+package instances
+
+import cats.effect.IO
+import cats.~>
+import com.twitter.util.{Future => TwitterFuture, Return, Throw}
+import scala.concurrent.{Future => ScalaFuture}
+import scala.util.{Failure, Success}
+
+trait ToIO {
```

Comment:
What's the reasoning for this being a trait? Why not move these two instances under `ToEffect` companion object?

Reply:
abstract them as trait so `ToEffect` could be more extensible, since `IO` is only one data type that implemented `Effect[_]`, there could be other i.e. monix's `Task`, so that we could easily just implement `Effect[Tasks]` as `trait ToTask` and
```scala
object ToEffect extends instances.ToIO with instances.ToTask
```

Reply:
Would it be shorter to generalize it for any `E : Effect` instead?

Reply:
🤔 good idea, these Future things just instances of `ToEffect[?, E]`